site/app/templates/calendar/Calendar.twig

```diff
@@ -68,24 +68,24 @@
                 </button>
                 <select id="calendar-item-type-edit" name="type" onchange="changeView(this);">
                     <option value="month" {% if (view_cookie is defined) and view_cookie == "month" %} selected {% endif %}>Month</option>
-                    <option value="two_week" {% if (view_cookie is defined)  and view_cookie == "two_week" %} selected {% endif %}>Two Week</option>
-                    <option value="one_week" {% if (view_cookie is defined)  and view_cookie == "one_week" %} selected {% endif %}>One Week</option>
+                    <option value="two_week" {% if (view_cookie is defined) and view_cookie == "two_week" %} selected {% endif %}>Two Week</option>
+                    <option value="one_week" {% if (view_cookie is defined) and view_cookie == "one_week" %} selected {% endif %}>One Week</option>
                 </select>
         </div>
 
         {# Legend #}
-        {% if show_legend == 1 %}
         <div class="cal-legend">
             {% if show_all_cookie == 1 %}
                 {% for course in course_names %}
-                    <div name="{{course.course_name|replace({' ':''})}}" class="legend-color"></div>
-                    <p for="{{course.course_name|replace({' ':''})}}-color" class="legend-label">{{course.display_name}}</p>
+                <div name="{{course.course_name|replace({' ':''})}}" class="legend-color" onclick="openOptionsModal()" style="cursor: pointer;" title="Click to open Calendar options form"></div>
+                <p for="{{course.course_name|replace({' ':''})}}-color" style="cursor: pointer;" onclick="filter_course('{{course.course_name}},{{course.display_name}}')" title="Click to filter course, click again to show all courses" class="legend-label">{{course.display_name}}</p>
+
```

Comment:
These lines are incredibly long.  Can you break them into multiple lines such that each HTML attribute is on a separate line?

Something like this perhaps?
```suggestion
                <div 
                    name="{{course.course_name|replace({' ':''})}}" 
                    class="legend-color" 
                    onclick="openOptionsModal()" 
                    style="cursor: pointer;" 
                    title="Click to open Calendar options form"
                ></div>
                <p 
                    for="{{course.course_name|replace({' ':''})}}-color" 
                    style="cursor: pointer;"
                    onclick="filter_course('{{course.course_name}},{{course.display_name}}')" 
                    title="Click to filter course, click again to show all courses" 
                    class="legend-label"
                >{{course.display_name}}</p>

```

Reply:
Please also move the inline CSS to an external stylesheet.